### PR TITLE
added statistics tab + checks by selenium

### PR DESCRIPTION
**Pull request by ynixon**

### CODE REVIEW

File: tests/navbar.py [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+from dotenv import load_dotenv
+from pathlib import Path
 import os
+import re
+import yaml
 from selenium import webdriver
 from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
 from selenium.common.exceptions import NoSuchElementException
 
+env_path = Path('..') / '.env'
```
Comment (lioramilbaum) [inline tests/navbar.py]: env_file is the name

File: app_s2i/templates/includes/_jumbotron.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="container">
   <div style="background:transparent !important" class="jumbotron jumbotron-fluid">
-    <h1 class="display-4">Welcome to the DevOps Learning Community</h1>
+    <h1 class="display-5">Welcome to the DevOps Learning Community</h1>
```
Comment (lioramilbaum) [inline app_s2i/templates/includes/_jumbotron.html]: The home page doesn't look good with this change. If you insist on adding it, verify it is applicable on all the rest, but the home page.

**Comment by lioramilbaum** [comment]
Another problem. See the attached screenshot.
Screen Shot 2019-06-04 at 7 11 50

File: app_s2i/templates/includes/_jumbotron.html (hunk 2 above)
Comment (ynixon) [reply app_s2i/templates/includes/_jumbotron.html]: rolled back

**Comment by ynixon** [comment]
> Another problem. See the attached screenshot.
> Screen Shot 2019-06-04 at 7 11 50

Changed the frame to be the same as the other tabs, now it looks better from my computer

File: tests/navbar.py (hunk 1 above)
Comment (ynixon) [reply tests/navbar.py]: env_file didn't work, leaving env_path

**Comment by lioramilbaum** [comment]
LGTM

**Pull request merged**